lms/djangoapps/philu_overrides/views.py

```diff
@@ -120,7 +120,7 @@ def login_and_registration_form(request, initial_mode="login", org_name=None, ad
         org_name = base64.b64decode(org_name)
         admin_email = base64.b64decode(admin_email)
 
-        email_field = registration_fields[2]
+        email_field = registration_fields[3]
```

Comment:
please finding the desired element by iterating over the list instead of using indexing